db/migrate/20240807160602_add_index_on_emails_id.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexOnEmailsId < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :emails, :id, algorithm: :concurrently
```

Comment:
Isn't there an automatic index on primary keys in PG?

Reply:
![image](https://github.com/user-attachments/assets/63c8e12d-74ef-44e7-95ac-a431cdf955c4)

Reply:
Good point - reading back through the `EXPLAIN` results I have, the index on the ID doesn't make much if any difference.
The other created_at index is where the benefits lie

Reply:
Removed in [14d15d0](https://github.com/DFE-Digital/apply-for-teacher-training/pull/9654/commits/14d15d0bb3d010ff6fdb944762eef7eb2203f814)